client/src/ui/molecules/login/index.tsx

```diff
@@ -72,10 +66,7 @@ function LoginInner() {
   return (
     <Dropdown id="user-avatar-menu" label={label} right={true} hideArrow={true}>
       <li>
-        <a href={viewProfileURL}>View profile</a>
-      </li>
-      <li>
-        <a href={editProfileURL}>Edit profile</a>
+        <a href={settingsURL}>Settings</a>
```

Comment:
Perhaps this should be `Account settings` instead?

Reply:
You are correct sir.

Reply:
Yes please. I still think "Settings" is shorter and better but it doesn't really matter because right now there's no home page for your accounts. 
I had it in my mind that we might soon have an accounts home page. And from there you might go deeper into your account settings.